writer.go

```diff
@@ -59,6 +59,11 @@ func (w *Writer) Write(b []byte) (int, error) {
 	return w.writeAndRetry(w.priority, string(b))
 }
 
+// RawWrite sends a log message with a custom priority
+func (w *Writer) RawWrite(p Priority, b []byte) (int, error) {
```

Comment:
Take a look at the `TestDebug`, `TestInfo`, etc functions in `writer_test.go`.

Those ones set up a writer with a default priority and then verify that they're able to send log messages with a priority different from that default. That's pretty much exactly what you want here.

So if you had a `TestWriteWithPriority` function that was structured very similarly to `TestDebug`, that would probably get you most of the way there.